src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const amount = days * 40;
+
+  if (days < 3) {
+    return amount;
+  } else if (days < 7) {
+    return amount - 20;
+  } else {
+    return amount - 50;
+  }
```

Comment:
try not use else if nesting. Just use early return
```suggestion
  if (days < 3) {
    return amount;
  }
  
  if (days < 7) {
    return amount - 20;
  } 
  
   return amount - 50;
```

Reply:
Fixed